pandas/tests/scalar/test_timedelta.py

```diff
@@ -9,9 +9,24 @@
 from pandas.core.tools.timedeltas import _coerce_scalar_to_timedelta_type as ct
 from pandas import (Timedelta, TimedeltaIndex, timedelta_range, Series,
                     to_timedelta, compat)
+from pandas.tseries.frequencies import to_offset
 from pandas._libs.tslib import iNaT, NaT
 
 
+class TestTimedeltaComparisons(object):
+    @pytest.mark.parametrize('freq', ['D', 'H', 'T', 's', 'ms', 'us', 'ns'])
+    def test_tick_comparison(self, freq):
+        offset = to_offset(freq) * 2
+        delta = offset.delta
+        assert isinstance(delta, Timedelta)
+        assert delta < offset
+        assert delta <= offset
+        assert not delta == offset
```

Comment:
these are not correct comparisons, eg.. ``not a == b`` is equiv to ``a != b`` and the linter complains about the former.

Reply:
If the linting is complaining I can change it to explicitly call e.g `not a.__ne__(b)`.  The methods have distinct implementations, need distinct tests.